http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/AppRoutes.scala

```diff
@@ -167,6 +175,25 @@ class AppRoutes(kubernetesService: AppService[IO], userInfoDirectives: UserInfoD
       resp <- ctx.span.fold(apiCall)(span => spanResource[IO](span, "listApp").use(_ => apiCall))
     } yield StatusCodes.OK -> resp
 
+  private[api] def updateAppHandler(userInfo: UserInfo,
+                                    googleProject: GoogleProject,
+                                    appName: AppName,
+                                    req: UpdateAppRequest
+  )(implicit
+    ev: Ask[IO, AppContext]
+  ): IO[ToResponseMarshallable] =
+    for {
+      ctx <- ev.ask[AppContext]
+      apiCall = kubernetesService.updateApp(userInfo, CloudContext.Gcp(googleProject), appName, req)
+      _ <- foldSpan(ctx, apiCall, "updateApp")
+    } yield StatusCodes.Accepted
+
+  // TODO: this exact pattern is repeated over 20x, and I have no idea what it does.  My intuition is that it's not
```

Comment:
I'm not planning to commit `foldSpan()` as-is.  Instead I would like a discussion about your norms / code standards.  Based on that, I'll either remove this or apply it more generally (perhaps in a different PR)